pydicom/config.py

```diff
@@ -71,6 +72,69 @@ def DS_decimal(use_Decimal_boolean=True):
 logger.addHandler(handler)
 
 
+image_handlers = []
+"""Image handlers for converting pixel data.
```

Comment:
@glemaitre Is there an easy way for me to see the docs generated by circleci from this build? It is not obvious from the control panel...